sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs

```diff
@@ -59,44 +58,45 @@ private static async Task Run(AppConfiguration configuration)
             {
                 LogicTokenProviderOptions tokenProviderOptions = new LogicTokenProviderOptions()
                 {
-                    AuthorizationScope = configuration.TokenProvider.AuthorizationScope, // "https://logicidentityprod.onmicrosoft.com/ac1d197a-0e7c-4add-83c5-e1b30a08efd6/.default",
-                    AuthorizationTokenIssuer = new Uri(configuration.TokenProvider.AuthorizationTokenIssuer.ToString()), //new Uri("https://login.microsoftonline.com/logicidentityprod.onmicrosoft.com/oauth2/v2.0/token"),
-                    ClientId = configuration.TokenProvider.ClientId, //"085d3847-1b71-4203-aa52-bbb98d5ce57c",
-                    ClientSecret = configuration.TokenProvider.ClientSecret // "Ox30ERiezE+gbat7k9jtCmnfKGISFoA8AVjnJo8IgH8="
+                    AuthorizationScope = configuration.TokenProvider.AuthorizationScope,
+                    AuthorizationTokenIssuer = new Uri(configuration.TokenProvider.AuthorizationTokenIssuer.ToString()),
+                    ClientId = configuration.TokenProvider.ClientId,
+                    ClientSecret = configuration.TokenProvider.ClientSecret,
                 };
                 configuration.Citizen.SubscriptionId = configuration.SubscriptionId;
                 configuration.Citizen.Serviceuri = configuration.Serviceuri;
                 var tokenProviderFactory = new LogicTokenProviderFactory(tokenProviderOptions);
                 var citizenDocumentClient = new CitizenDocumentsClient(httpClient, tokenProviderFactory, configuration.Citizen);
-                var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfiguartionId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
+                var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfigurationId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
 
                 if (uploadDocument == null)
                 {
-                    Log.Error("There is error occured in upload");
+                    Log.Error("An error occurred while document upload operation");
```

Comment:
bad grammar. 

I suggest something like: 

> An unknown error occurred during the document upload

Furthermore, why should this return `null`? Isn't null a bad response, since it's very difficult for end users to understand what it means, and they have to explicitly check for it?

Reply:
Ok corrected as per your input.

Reply:
I didn't get a response to the question about returning null

Reply:
Ahh i see. Mistakenly this was pushed in the code branch. However here upload document API does not return Not found. So removed it now